src/test/scala/examples/GCDSpec.scala

```diff
@@ -88,7 +88,7 @@ class GCDSpec extends FlatSpec with Matchers {
 
   it should "run verilator via command line arguments" in {
     // val args = Array.empty[String]
-    val args = Array("--backend-name", "verilator")
+    val args = Array("--backend-name", "gui")
```

Comment:
Is this intentional? What kind of backend is "gui"? Should the test name be changed to:
```scala
it should "run gui via command line arguments" in {
```